Week3/Homework-Week3/assets/morejs_5.js

```diff
@@ -0,0 +1,15 @@
+{
+  /*
+Can you write the following without the if statement, 
+but with just as a single line with console.log(...);?
+
+
+if (3 === 3) {
+  console.log('yes');
+} else {
+  console.log('no');
+}
+*/
+
+  console.log("No, I couldn't find a way to replace if statement with single line of console log");
```

Comment:
Ahh, did you look into the [conditional operator](https://www.w3schools.com/js/js_comparisons.asp). Maybe that will do the trick:)